vscode-extension/src/extension.ts

```diff
@@ -26,6 +32,46 @@ export function activate(context: vscode.ExtensionContext) {
   });
   context.subscriptions.push(setupCommand);
 
+  let shareCommand = vscode.commands.registerCommand(COMMANDS.SHARE, async () => {
+    // TODO: use vscode.window.activeTextEditor instead of hardcoding
+    const fileName: string = "/Users/rossdancraig/Projects/aiconfig/cookbooks/Gemini/gemini_demo.aiconfig.json";
+    const sanitizedFileName: string = sanitizeFileName(fileName);
+
+    // Use empty accessKeyId and secretAccessKey so bucket can be accessed publicly
+    AWS.config.update({
+      accessKeyId: '',
+      secretAccessKey: '',
+    });
+    const s3 = new AWS.S3();
```

Comment:
Does this need any handling to work without aws credentials (as in https://github.com/lastmile-ai/gradio-workbook/pull/153/files)?

Reply:
Oh, ok. Can you please update the comment to make that a bit clear (specifically noting 'without credentials')